pontoon/insights/static/js/insights.js

```diff
@@ -245,6 +246,109 @@ var Pontoon = (function (my) {
           },
         });
       },
+      renderTimeToReviewPretranslatons: function () {
+        var chart = $('#time-to-review-pretranslations-chart');
```

Comment:
Should prefer `const`. Don't we have a linter rule for that?

Reply:
Am confused by the 👍 reaction. So is anything happening here about this?

Reply:
Removed. :-)

When we introduced linters and formatters, the main argument was to let the tools style the code and speed up the review process. I suggest we file an issue to implement the linter rule if that's possible and keep that as is until then. There is a lot of vars in our codebase.

Reply:
Added #2931.